src/handlers/issue-matching.ts

```diff
@@ -0,0 +1,150 @@
+import { Context } from "../types";
+import { IssuePayload } from "../types/payload";
+
+export interface IssueGraphqlResponse {
+  node: {
+    title: string;
+    url: string;
+    state: string;
+    stateReason: string;
+    closed: boolean;
+    repository: {
+      owner: {
+        login: string;
+      };
+      name: string;
+    };
+    assignees: {
+      nodes: Array<{
+        login: string;
+        url: string;
+      }>;
+    };
+  };
+  similarity: number;
+}
+
+const commentBuilder = (matchResultArray: Map<string, Array<string>>): string => {
+  const commentLines: string[] = [">[!NOTE]", ">The following contributors may be suitable for this task:"];
```

Comment:
```suggestion
  const commentLines: string[] = ["> [!NOTE]", "> The following contributors may be suitable for this task:"];
```

Reply:
Adding the space, somehow breaks it. I tried without space, it does not render the note.